package.json

```diff
@@ -17,7 +17,8 @@
     "test:watch": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wtr --watch\"",
     "deploy": "yarn run deploy:run -e $(git branch --show-current)",
     "deploy:run": "yarn run prepare:ghpages && touch ghpages/.nojekyll && yarn run deploy:gh",
-    "deploy:gh": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\""
+    "deploy:gh": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\"",
+    "typecheck": "node node_modules/typescript/bin/tsc --noEmit"
```

Comment:
what's the difference with `npx tsp --noEmit`?

Reply:
u think we should add typecheck to test script?

Reply:
`yarn tsc --noEmit` can be simplified to that - we can add it too in test script

Reply:
pardon, `tsc` is already in `test` script.

Reply:
updated package.json